src/main/java/codeu/model/data/User.java

```diff
@@ -19,45 +19,91 @@
 
 /** Class representing a registered user. */
 public class User {
-  private final UUID id;
-  private final String name;
-  private final String password;
-  private final Instant creation;
-
-  /**
-   * Constructs a new User.
-   *
-   * @param id the ID of this User
-   * @param name the username of this User
-   * @param password the password of this User
-   * @param creation the creation time of this User
-   */
-  public User(UUID id, String name, String password, Instant creation) {
-     this.id = id;
-     this.name = name;
-     this.password = password;
-     this.creation = creation;
-  }
-
-  /** Returns the ID of this User. */
-  public UUID getId() {
-    return id;
-  }
-
-  /** Returns the username of this User. */
-  public String getName() {
-    return name;
-  }
-
-  /**
-   * Returns the password of this User.
-   */
-  public String getPassword() {
-    return password;
-  }
-
-  /** Returns the creation time of this User. */
-  public Instant getCreationTime() {
-    return creation;
-  }
+    private final UUID id;
+    private final String name;
+    private final String password;
+    private final Instant creation;
+    private final Integer Age;
+    private final String firstName;
+    private final String lastName;
+    private final String email;
+    private final String phoneNum;
+    private final String bio;
+
+    /**
+     * Constructs a new User.
+     *
+     * @param id
+     *            the ID of this User
```

Comment:
Please keep the documentation like it was before

@param id the ID of this User
@param name the username of this User
...
...